index.html

```diff
@@ -197,103 +323,111 @@ <h2>About me</h2>
             hestiate to contact me.
           </p>
           <button class="about-button" type="button">Get my Resume</button>
```

Comment:
The container of this button find it hard to hold the text inside it:
Your design:
![image](https://user-images.githubusercontent.com/83596135/160215937-a45a39ac-e68c-4f0c-b437-e95f16ae3a5c.png)

- [ ] Kindly, fix this issue in order to have a nice looking portfolio :+1: